schedule/yast/autoyast/autoyast_create_hdd_sdk_gnome.yaml

```diff
@@ -20,6 +20,7 @@ schedule:
   - '{{handle_reboot}}'
   - installation/first_boot
   - console/system_prepare
+  - '{{disable_screensaver}}'
```

Comment:
I think you make it mixed with another PR, please clean it.

Reply:
Fixed, but I don't why there are two commits, I've rebased it.